tests/test_local.rs

```diff
@@ -59,13 +59,19 @@ async fn local_smoke_test() -> Result<(), Box<dyn std::error::Error>> {
     // Make sure we can push LFS objects to the server.
     repo.lfs_push()?;
 
-    // Make sure we can re-download the same objects.
-    repo.clean_lfs()?;
-    repo.lfs_pull()?;
-
     // Push again. This should be super fast.
     repo.lfs_push()?;
 
+    // This should be fast since we already have the data
+    repo.lfs_pull()?;
+
+    // Make sure we can re-download the same objects in another repo
+    let repo2 = GitRepo::init(addr)?;
+    repo2.lfs_pull()?;
```

Comment:
Since this is a new empty repo with no LFS objects, I don't think this will download anything. I'm not really sure what you're trying to accomplish?